front/app/components/FiltersModal/TopBar.tsx

```diff
@@ -11,14 +11,21 @@ import messages from './messages';
 import styled from 'styled-components';
 import { media, colors, fontSizes } from 'utils/styleUtils';
 import { lighten } from 'polished';
-import { ScreenReaderOnly } from 'utils/a11y';
 
 const Container = styled.div`
   height: ${(props) => props.theme.mobileTopBarHeight}px;
   background: #fff;
   border-bottom: solid 1px ${lighten(0.4, colors.label)};
 `;
 
+const StyledCloseIconButton = styled(CloseIconButton)`
```

Comment:
Not super important but with the Box props added to the IconButton you should be able to do this without the need for a styled component.